R/zzz.R

```diff
@@ -3,6 +3,7 @@ registered_handlers_namespaces <- new.env()
 
 .onLoad <- function(libname, pkgname) { # nolint
   # Set up the teal logger instance
+  teal_logger_formatter()
```

Comment:
just to make sure it's not part of the `teal.logger` package' instance of logger object

```suggestion
  teal_logger_formatter()
  # Set up the teal logger instance
```